javascript-functions/main.js

```diff
@@ -0,0 +1,24 @@
+function convertMinutesToSeconds(minutes) {
+  console.log('value of totalMinutes: ', minutes * 60);
```

Comment:
you don't want to have any console logs in the functions, the functions should return the answer and then you store it in a variable, then log the variable.